packages/block-library/src/gallery/tiles/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Children } from '@wordpress/element';
+
+/**
+ * WordPress dependencies
+ */
+
+function Tiles( props ) {
+	const {
+		className,
+		tilesProps: {
+			align,
+			columns,
+			imageCrop,
+		},
+		children,
+	} = props;
+
+	const newClassName = classnames(
+		'wp-tiles',
```

Comment:
Following the [CSS Coding Guidelines](https://github.com/WordPress/gutenberg/blob/713807dee2378951678d5971ff76d4a58d39c7d0/docs/contributors/coding-guidelines.md#css), I'm not 100% sure about the naming, since this is a component inside a block, but I think the class name would be `wp-block-gallery-tiles` and each item `wp-block-gallery-tiles__tile`